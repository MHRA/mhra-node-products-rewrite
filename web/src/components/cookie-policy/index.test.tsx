import renderer from 'react-test-renderer';
import React from 'react';
import CookieBanner from './index';

describe(CookieBanner, () => {
  it('should render 🍪', () => {
    // tslint:disable-next-line: no-empty
    const noop = () => {};
    const component = renderer
      .create(<CookieBanner storageAllowed setStorageAllowed={noop} />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
