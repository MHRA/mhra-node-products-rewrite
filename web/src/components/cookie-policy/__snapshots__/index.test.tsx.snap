// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CookieBanner should render 🍪 1`] = `" "`;
